Bottom: 797b1fd2aad227cfa8e32dd2681791e8d7d53be1
Top:    67662b07acae23266cacafe72a86e121674c00af
Author: Ian Kent <ikent@redhat.com>
Date:   2013-07-24 16:01:55 +0800

Refresh of autofs-5.0.7-dont-override-LDFLAGS-in-make-rules.patch

---

diff --git a/autofs.spec b/autofs.spec
index bcc3831..9a2bf6e 100644
--- a/autofs.spec
+++ b/autofs.spec
@@ -76,7 +76,7 @@ inkludera n
 %setup -q
 echo %{version}-%{release} > .version
 %if %{with_systemd}
-  %define unitdir %{?_unitdir:/usr/lib/systemd/system}
+  %define unitdir %{?_unitdir:/lib/systemd/system}
   %define systemd_configure_arg --with-systemd
 %endif
 %if %{with_libtirpc}
@@ -91,7 +91,7 @@ LDFLAGS="-Wl,-z,now" \
 	--enable-ignore-busy \
 	%{?systemd_configure_arg:} \
 	%{?libtirpc_configure_arg:}
-CFLAGS="$RPM_OPT_FLAGS -Wall" make initdir=/etc/rc.d/init.d DONTSTRIP=1
+CFLAGS="$RPM_OPT_FLAGS -Wall" LDFLAGS="-Wl,-z,now" make initdir=/etc/rc.d/init.d DONTSTRIP=1
 
 %install
 rm -rf $RPM_BUILD_ROOT
